test/node/decimal128.test.ts

```diff
@@ -210,8 +212,8 @@ describe('Decimal128', function () {
 
   it('fromString scientific format', function (done) {
     // Create decimal from string value 10e0
-    var result = Decimal128.fromString('10e0');
-    var bytes = Buffer.from(
+    let result = Decimal128.fromString('10e0');
```

Comment:
Same here for splitting out multiple `it` blocks as well as all the subsequent tests.